src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
 // write code here
+
+const employees = document.getElementsByTagName('li');
+
+function sortList(toBeSorted) {
+  const employeesArray = [...toBeSorted].sort((a, b) =>
+    b.dataset.salary.substring(1).replaceAll(',', '')
+  - a.dataset.salary.substring(1).replaceAll(',', ''));
+
+  const StartPoint = document.getElementsByTagName('ul');
```

Comment:
Consider using querySelector or querySelectorAll instead of getElementsByTagName for better performance and compatibility.